make/autoconf/help.m4

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2011, 2022, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2011, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please revert the copyright change to keep the original (2011, 2022).

Reply:
OK, I've reverted the copyright change.